libs/ui/src/lib/datepicker/datepicker.component.html

```diff
@@ -0,0 +1,36 @@
+<div
+  class="cvi-datepicker__input-container"
+  tabindex="0"
+  [ngClass]="{ 'cvi-datepicker__input-container-focused': focus }"
+  (focus)="setFocus(true)"
+  [attr.aria-expanded]="focus"
+>
+  <label [attr.for]="htmlId" class="cvi-datepicker__calendar-visually-hidden">Select Date</label>
+  <input
+    type="text"
```

Comment:
it would be great to give an accessible description of the date format to the input( could be done with aria-describedby)